website_sale_product_minimal_price/models/product_template.py

```diff
@@ -91,24 +96,49 @@ def _get_first_possible_combination(
         res = super()._get_first_possible_combination(
             parent_combination=parent_combination, necessary_values=necessary_values
         )
-        context = self.env.context
-        if (
-            context.get("website_id")
-            and context.get("pricelist")
-            and self.product_variant_count > 1
-        ):
+        # Copied from _get_combination_info
+        # /odoo/addons/website_sale/models/product_template.py
+        website = (
+            # pylint: disable=W8121
```

Comment:
Same here, don't silence the linter, but follow its hints

Reply:
Please check comment above. I just search a current website in the same way as it made in Odoo CE
https://github.com/odoo/odoo/blob/dd97f1aa0b5a0f4e87be663cc7f2f610ee052e4b/addons/website_sale/models/product_template.py#L392
But current linter does not allow get in in the same way, so I've disabled it

Reply:
Fixed